pkg/network/node/ovscontroller.go

```diff
@@ -97,9 +96,7 @@ func (oc *ovsController) SetupOVS(clusterNetworkCIDR []string, serviceNetworkCID
 	otx := oc.ovs.NewTransaction()
 
 	// Table 0: initial dispatch based on in_port
-	if oc.useConnTrack {
-		otx.AddFlow("table=0, priority=300, ip, ct_state=-trk, actions=ct(table=0)")
-	}
+	otx.AddFlow("table=0, priority=300, ip, ct_state=-trk, actions=ct(table=0)")
```

Comment:
oh, while we're changing this anyway, there's a bug here: this was supposed to be the highest-priority rule in table 0, but there's a `priority=400` rule below. So can you bump this to `priority=1000`?